frontend/src/Home.js

```diff
@@ -23,10 +26,20 @@ const TEST_QUERY = gql`
 export default function Home() {
   const { loading, error, data } = useQuery(TEST_QUERY);
 
+  const { isAuthenticated, loginWithRedirect, logout } = useAuth0();
+
   return (
     <div>
       <h1>Home</h1>
+      <Link to="/profile">Profile</Link>
       <pre>{JSON.stringify(data, null, 2)}</pre>
+      <div>
+        {!isAuthenticated && (
+          <button onClick={() => loginWithRedirect({})}>Log in</button>
```

Comment:
Kannst ruhig deutsch arbeiten

```suggestion
          <button onClick={() => loginWithRedirect({})}>Anmelden</button>
```